spec/aruba/matchers/file_spec.rb

```diff
@@ -135,18 +135,43 @@ def fail_with(message)
       let(:reference_file_content) { 'bar' }
 
       context 'and this is expected' do
-        it { expect(file_name).not_to have_same_file_content_like reference_file }
+        it { expect(file_name).not_to have_same_file_content_as reference_file }
       end
 
       context 'and this is not expected' do
         it do
-          expect { expect(file_name).to have_same_file_content_like reference_file }
+          expect { expect(file_name).to have_same_file_content_as reference_file }
             .to raise_error RSpec::Expectations::ExpectationNotMetError
         end
       end
     end
   end
 
+  describe "to have_same_file_content_like" do
+    it 'calls have_same_file_content_as' do
+      expect(self).to receive(:have_same_file_content_as)
```

Comment:
Can we test this some other way? Mocking `self` can lead to all kinds of breakage down the line.

I think you can just test these matchers by using them as intended (though less extensively than the main tests above).

Reply:
Yep, I'll change this over, too. Thanks for reviewing! :)

Reply:
You're welcome!